copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,101 @@ main loop frequency divided by a sub-sample factor.
 | 0            |    0         | 8             |
 +--------------+--------------+---------------+
 
+Identification of a Multicopter
+===============================
+Description of the Approach
+---------------------------
+
+The Parameter :ref:`SID_AXIS<SID_AXIS>` of the system identification mode feeds in the frequency-sweep at different spots inside Ardupilot's controlers.
+The locations of these injection points are shown in the following :ref:`figure<fig-ctrl-sys-ardupilot>` as small chirp symbols.
+At each injection point, the sweep can be added to either the roll, pitch or yaw axis. 
+
+.. image:: ../images/ControlSystem.png
+:name: fig-ctrl-sys-ardupilot
+
+For the purpose of obtaining a dynamic model of only the copter itself (aka plant model), it is first necessary to calculate a transfer function(s) between the control inputs of the copter and its measurable dynamic response.
+In this case, the angular rate controller outputs (:math:`p_{Thrust}`, :math:`q_{Thrust}` and :math:`r_{Thrust}`) represent the control inputs of the copter whereas the gyroscope's measured angular rates represent the system response.
+Since a common application of the created system model consists in optimizing the control parameters, the filtered gyroscope signals are used instead of the raw gyroscope data, as these are the direct inputs to the angular rate controller.
```

Comment:
I don't necessarily agree with this approach.  Using the filtered gyroscope signals can cause issues if this model was used in SITL or in any other simulation environment when the controller expects raw gyroscopic data.  You are using simulink to wrap a controller around this flight dynamic model, why not just add the filters to the controller and use the unfiltered gyroscopic data to develop the raw flight dynamic model.